src/components/client/campaign-application/CampaignApplicationPage.tsx

```diff
@@ -1,10 +1,24 @@
+import { CircularProgress, Grid } from '@mui/material'
+import { useSession } from 'next-auth/react'
+import { useViewPersonByKeylockId } from 'common/hooks/person'
 import Layout from '../layout/Layout'
 import CampaignApplicationForm from './CampaignApplicationForm'
 
 export default function CampaignApplicationPage() {
+  const { data: session } = useSession()
+  const { data: person, isLoading } = useViewPersonByKeylockId(session?.user?.sub as string)
```

Comment:
Check useCurrentPerson hook, should do the exact same job.

Reply:
Yep, done